src/SALib/test_functions/Sobol_G.py

```diff
@@ -26,40 +72,48 @@ def evaluate(values, a=None):
 
     Y = np.ones([values.shape[0]])
 
-    len_a = len(a)
     for i, row in enumerate(values):
-        for j in range(len_a):
-            x = row[j]
-            a_j = a[j]
-            Y[i] *= (np.abs(4 * x - 2) + a_j) / (1 + a_j)
+        shift_of_x = row + delta
+        integral = np.modf(shift_of_x)[1]
+        mod_x = shift_of_x - integral
+        temp_y = (np.abs(2 * mod_x - 1)**alpha)
+        y_elements = ((1 + alpha) * temp_y + a) / (1 + a)
+        Y[i] = np.prod(y_elements)
 
     return Y
 
 
-def partial_first_order_variance(a=None):
+
+def partial_first_order_variance(a=None, alpha=None):

```

Comment:
If this is a private function e.g. not intended to be called outside of the module, could you prepend `_` e.g. `def _partial_first_order_variance()`